tests/parm/input.benchmark_ccpp.nml.IN

```diff
@@ -0,0 +1,321 @@
+&atmos_model_nml
```

Comment:
This file should be removed. It is no longer used and was removed in PR #623

Reply:
The '&atmos_model_nml' section was removed.